### PR TITLE
Field config data size

### PR DESCRIPTION
**Pull request by joakim-hove**
**Issue**
Small PR in the localisation work

**Approach**
Will expose the data size of a field in the `FieldConfig` class

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

**Comment by ManInFez** [comment]
Test this please!

File: python/res/enkf/config/field_config.py [hunk 1]
```diff
@@ -107,6 +110,9 @@ def get_ny(self):
     def get_nz(self):
         return self._get_nz()
 
+    def data_size(self):
```
Comment (ManInFez) [inline python/res/enkf/config/field_config.py]: this could be `get_data_size()` as all the other functions are `get_`

**Comment by ManInFez** [comment]
Just a small comment, squash and rebase, and you are good to go :)

**Comment by joakim-hove** [comment]
OK; name changed `data_size()` &rightarrow; `get_data_size()` and squashed to one commit.

**Comment by ManInFez** [comment]
Test this please!

**Review by ManInFez — APPROVED** [review]
🥓

**Pull request merged**